catalog/core/templates/dashboard.html

```diff
@@ -73,7 +73,7 @@ <h3 class="panel-title">Recently Updated Publications</h3>
                                 {% endif %}
                                 </td>
                                 <td>{%  for key, value in contribution.items %}
-                                    {%  if key == pub.id %}
+                                    {%  if key == contribution_cache_prefix|addstr:pub.id %}
```

Comment:
I'm not a fan of this kind of munging. Is it possible to either change `contribution.items` to remove the cache prefix from the key or see if it's possible to use https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#in-operator, e.g., `if pub.id in key` - I think that might cause false positives however:

 `'2170' in '12170' == True`